.circleci/cluster/validate_status.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+set -eux
+
+validate_status() {
+  manager_status="$(docker exec $1 cfy cluster status)"
+  set +e
+  grep --ignore-case -q -e 'fail' -e 'error' <<< "$manager_status" && (echo "Found a failure in the cluster status" && exit 1)
```

Comment:
you can change it to a little shorter command 'grep -iq'

Reply:
200 iq comment